bin/jruby.sh

```diff
@@ -473,6 +482,11 @@ dir_name "$JAVACMD"
 dir_name "$REPLY"
 JAVA_HOME="$REPLY"
 
+if ! [ -e "$JAVA_HOME/release" ]; then
```

Comment:
This is fine for JRuby 10 but it will not work with JRuby 9.4 when we backport the script.

Reply:
Your most recent changes handle this better so I removed this